resources/extracted_flagart_bitmaps/README.rst

```diff
@@ -0,0 +1,322 @@
+wx/lib/art/flagart PNGs
+=======================
+
+These images have been optimized with FileOptimizer http://nikkhokkho.sourceforge.net/static.php?page=FileOptimizer
+
+
+TODO
+----
+ * map every flag with a reference for sanity.
+
+
+BlaBla
+------
+"""
+See: http://www.famfamfam.com/lab/icons/flags
+
+FamFamFam Flag icons are available free for any purpose
+Copyright Mark James (mjames@gmail.com)
+
+This module is (c) 2006 Riaan Booysen and licensed under the Python license.
+
+"""
+
+
+Sanity checks
+-------------
+ * https://en.wikipedia.org
+ * http://flagpedia.net/
+
+
++--------------------------+------------+-------------------------------------------+--------------------------------------------------------------------------------------------------+
+| IMG                      | PNG        | Country Name                              | Flag Reference                                                                                   |
```

Comment:
Please reduce the width of this table to the table readable as text in a window that is 100 characters wide, or at least as close to 100 as you can get it.